fidelisnetwork.json

```diff
@@ -0,0 +1,1073 @@
+                    "description": "Specify the maximum number of alerts to return. You can specify between 1 and 200,000. (Default is 100)",
+                    "data_type": "numeric",
+                    "default": 100,
+                    "order": 4
+                }
+            },
+            "render": {
+                "type": "table",
+                "width": 12,
+                "height": 5,
+                "title": "List Alerts"
+            },
+            "output": [
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.total_alerts",
+                    "example_values": [
+                        512
+                    ],
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.data.*.aaData.*.ALERT_ID",
+                    "example_values": [
+                        false
+                    ],
+                    "column_name": "Alert ID",
+                    "column_order": 0,
+                    "data_type": "string",
+                    "contains": [
+                        "alert id"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.aaData.*.SEVERITY",
+                    "example_values": [
+                        false
+                    ],
+                    "column_name": "SEVERITY",
+                    "column_order": 1,
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.aaData.*.HOST_IP",
+                    "example_values": [
+                        false
+                    ],
+                    "column_name": "HOST IP",
+                    "column_order": 2,
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.aaData.*.ALERT_TYPE",
+                    "example_values": [
+                        false
+                    ],
+                    "column_name": "ALERT TYPE",
+                    "column_order": 3,
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.aaData.*.ALERT_TIME",
+                    "example_values": [
+                        false
+                    ],
+                    "column_name": "ALERT TIME",
+                    "column_order": 4,
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.aaData.*.SUMMARY",
+                    "example_values": [
+                        false
```

Comment:
Should the value here be some texts instead of boolean?

Reply:
Yes, The example value should be summary of alert. I updated it.